### PR TITLE
[Spanish] Translate guides configuration glossary

### PR DESCRIPTION
**Pull request by kinxos**
Pages translated to Spanish for #543

📝 guides -> configuration-glossary

- [x] configuration-hooks.md
- [x] configuration-ignore.md
- [x] configuration-loading.md
- [x] configuration-loading-indicator.md

### CODE REVIEW

**Comment by debs-obrien** [comment]
@Dawntraoz @alvarosaburido can either of you give this a quick review please

**Comment by kinxos** [comment]
@Dawntraoz @alvarosaburido There's two words particularly that I didn't know how to translate, so I didn't: build (as in 'the build process') and legacy (as in 'legacy bundle').

**Comment by alvarosaburido** [comment]
> @Dawntraoz @alvarosaburido There's two words particularly that I didn't know how to translate, so I didn't: build (as in 'the build process') and legacy (as in 'legacy bundle').

Yeah, there are specific terms that lose meaning inside of software development when translate it to spanish:

- Build process literally translated would be, Proceso de construcción, which is not used in the spanish IT world. You often hear "El proceso de _build_ no funciono" 

- Legacy code seria: "Codigo heredado" which is not really the meaning of the term, we used for old code.

So I would use the english terms, but in italic to highlight that are english terms.

**Comment by kinxos** [comment]
> > @Dawntraoz @alvarosaburido There's two words particularly that I didn't know how to translate, so I didn't: build (as in 'the build process') and legacy (as in 'legacy bundle').
> 
> Yeah, there are specific terms that lose meaning inside of software development when translate it to spanish:
> 
> - Build process literally translated would be, Proceso de construcción, which is not used in the spanish IT world. You often hear "El proceso de _build_ no funciono" 
> 
> - Legacy code seria: "Codigo heredado" which is not really the meaning of the term, we used for old code.
> 
> So I would use the english terms, but in italic to highlight that are english terms. 

Awesome! I'll change those terms to italic then. Thanks for the advice!

**Comment by kinxos** [comment]
Texts have been updated, with italized non-translatable words.

File: content/es/guides/configuration-glossary/configuration-hooks.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+- [_Hooks_s de Builder](/guides/internals-glossary/internals-builder#hooks)
+- [_Hooks_s de Generator](/guides/internals-glossary/internals-generator#hooks)
+
+## Ejemplos
+
+### Redirigir a router.base cuando no se está en root
+
+Imaginemos que quieres servir páginas como `/portal` en lugar de `/`.
+
+Quizá éste sea un caso extremo, y además el sentido del `router.base` de _nuxt.config.js_ es para cuando un servidor web sirve Nuxt fuera de la ruta del dominio.
```
Comment (alvarosaburido) [inline content/es/guides/configuration-glossary/configuration-hooks.md:48]: Quizá éste sea un caso extremo, además el propósito del `router.base` en el _nuxt.config.js_ sirve cuando un servidor web usa Nuxt fuera de la ruta principal del dominio.

File: content/es/guides/configuration-glossary/configuration-hooks.md [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+
+Pero en desarrollo local, _localhost_, cuando router.base no es /, devuelve un 404. Para evitar esto, puedes definir un _hook_.
```
Comment (alvarosaburido) [inline content/es/guides/configuration-glossary/configuration-hooks.md:50]: Pero en desarrollo local, al acceder a  _localhost_ cuando router.base no es / devolverá un 404. Para evitar esto, puedes definir un _hook_.

File: content/es/guides/configuration-glossary/configuration-hooks.md [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+         res.writeHead(302, {
+           Location
+         })
+         res.end()
+       }
+       next()
+     }
+   ```
+
+Después, cuando un compañero navegue sin querer a `/`, Nuxt redireccionará automáticamente a `/portal`
```
Comment (alvarosaburido) [inline content/es/guides/configuration-glossary/configuration-hooks.md:147]: Después, cuando un compañero de desarrollo navegue sin querer a `/`, Nuxt redireccionará automáticamente a `/portal`

File: content/es/guides/configuration-glossary/configuration-ignore.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+title: 'La propiedad ignore'
+description: Define los archivos a ignorar para tu aplicación Nuxt.js
+menu: ignore
+category: configuration-glossary
+position: 14
+---
+
+## .nuxtignore
+
+Puedes usar un fichero `.nuxtignore` para permitir que Nuxt.js ignore ficheros `layout`, `page`, `store` y `middleware` en el directorio raíz de tu proyecto (`rootDir`) durante la fase _build_. El fichero `.nuxtignore` sigue la misma especificación que `.gitignore` y `.eslintignore`, según la cual cada línea es un patrón glob, que indica qué ficheros deberían ser ignorados.
```
Comment (alvarosaburido) [inline content/es/guides/configuration-glossary/configuration-ignore.md:11]: 1. Puedes usar el fichero `.nuxtignore` para permitir que Nuxt.js ignore los archivos `layout`, `page` ... dentro del directorio raíz de tu proyecto (`rootDir`)
2. cada línea es un patrón _glob_

File: content/es/guides/configuration-glossary/configuration-ignore.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+> Cualquier fichero en pages/, layouts/, middleware/ o store/ será ignorado durante la fase build si su nombre empieza con el prefijo especificado por `ignorePrefix`.
+
+Por defecto, todos los ficheros que empiecen por `-` serán ignorados, como por ejemplo `store/-foo.js` y `pages/-bar.vue`. Esto habilita tests de co-localización, utilidades, y componentes con sus callers, sin que ellos mismos sean convertidos a routes, stores, etc.
+
+## La propiedad ignore
+
+- Tipo: `Array`
+- Por defecto: `['**/*.test.*']`
+
+> Más personalizable que `ignorePrefix`: todos los ficheros que coincidan con los patrones glob definidos en `ignore` serán ignorados en la fase build.
```
Comment (alvarosaburido) [inline content/es/guides/configuration-glossary/configuration-ignore.md:52]: _glob_

File: content/es/guides/configuration-glossary/configuration-ignore.md [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+
+## ignoreOptions
+
+`nuxtignore` usa `node-ignore` bajo el capó, `ignoreOptions` puede ser configurado como `options` de `node-ignore`.
```
Comment (alvarosaburido) [inline content/es/guides/configuration-glossary/configuration-ignore.md:56]: `nuxtignore` usa `node-ignore` por debajo, `ignoreOptions` puede ser configurado como `options` de `node-ignore`.

la expresión "bajo el capó" es una traducción literal de "under the hood" pero no es usada frecuentemente en castellano

File: content/es/guides/configuration-glossary/configuration-loading-indicator.md [hunk 7]
```diff
@@ -0,0 +1,47 @@
+---
+title: 'La propiedad loading indicator'
+description: ¡Muestra un indicador de carga chulo mientras se carga la SPA!
```
Comment (alvarosaburido) [inline content/es/guides/configuration-glossary/configuration-loading-indicator.md:3]: Depending on the Hispanic-speaking country, "Chulo" can have a negative connotation. Please remove it. 

https://dle.rae.es/chulo

File: content/es/guides/configuration-glossary/configuration-loading.md [hunk 8]
```diff
@@ -0,0 +1,117 @@
...
+```
+
+Lista de propiedades para personalizar la barra de progreso.
+
+| Clave | Tipo | Defecto | Descripción |
+| --- | --- | --- | --- |
+| `color` | String | `'black'` | Color CSS de la barra |
+| `failedColor` | String | `'red'` | Color CSS de la barra cuando se produjo un error durante el renderizado de la ruta (si `data` o `fetch` dieron error, por ejemplo). |
+| `height` | String | `'2px'` | Altura de la barra (se usa en la propiedad `style` de la barra) |
+| `throttle` | Number | `200` | En ms, indica la espera antes de mostrar la barra. Útil para prevenir el parpadeo de la barra. |
```
Comment (alvarosaburido) [inline content/es/guides/configuration-glossary/configuration-loading.md:55]: | `throttle` | Number | `200` | En ms, indica la espera antes de mostrar la barra. Útil para prevenir el parpadeo de la misma. |

**Review by alvarosaburido — CHANGES_REQUESTED** [review]
@kinxos Hi! I left some comments to be review. Overall great work

**Comment by kinxos** [comment]
Genial @alvarosaburido Voy a por los cambios!

File: content/es/guides/configuration-glossary/configuration-hooks.md (hunk 1 above)
Comment (kinxos) [reply content/es/guides/configuration-glossary/configuration-hooks.md]: Hecho!

File: content/es/guides/configuration-glossary/configuration-hooks.md (hunk 2 above)
Comment (kinxos) [reply content/es/guides/configuration-glossary/configuration-hooks.md]: Hecho!

File: content/es/guides/configuration-glossary/configuration-hooks.md (hunk 3 above)
Comment (kinxos) [reply content/es/guides/configuration-glossary/configuration-hooks.md]: Hecho!

File: content/es/guides/configuration-glossary/configuration-ignore.md (hunk 4 above)
Comment (kinxos) [reply content/es/guides/configuration-glossary/configuration-ignore.md]: Hecho!

File: content/es/guides/configuration-glossary/configuration-ignore.md (hunk 5 above)
Comment (kinxos) [reply content/es/guides/configuration-glossary/configuration-ignore.md]: Hecho!

File: content/es/guides/configuration-glossary/configuration-ignore.md (hunk 6 above)
Comment (kinxos) [reply content/es/guides/configuration-glossary/configuration-ignore.md]: Tienes razón, pero tampoco me sonaba demasiado bien la expresión "por debajo", aunque me imagino que es más genérica y, por tanto, mejor entendida. Hecho!

File: content/es/guides/configuration-glossary/configuration-loading-indicator.md (hunk 7 above)
Comment (kinxos) [reply content/es/guides/configuration-glossary/configuration-loading-indicator.md]: No caí en esto. Lo sustituí por "atractivo". Hecho!

File: content/es/guides/configuration-glossary/configuration-loading.md (hunk 8 above)
Comment (kinxos) [reply content/es/guides/configuration-glossary/configuration-loading.md]: Hecho!

**Comment by debs-obrien** [comment]
Great teamwork guys and thanks for your contributions 💪

**Pull request merged**